src/components/Table/Table.styles.ts

```diff
@@ -0,0 +1,20 @@
+import styled from 'styled-components/macro';
+
+import { Card as MuiCard, Chip as MuiChip } from '@material-ui/core';
+
+import { spacing } from '@material-ui/system';
+
+export const Card = styled(MuiCard)(spacing);
+
+export const Chip = styled(MuiChip)<{ rgbcolor: string }>`
+  height: 20px;
+  padding: 4px 0;
+  font-size: 90%;
```

Comment:
I think to it will be good to change `rem` units and make it consistent with all places now in theme where `px` or `%` are used. I will create another ticket for this. It will be easier to change it in future if we will need to change font-sizes globally

Reply:
Sure, good idea.